specification.markdown

```diff
@@ -320,7 +320,7 @@ The path defines the allowed storage locations of the `EBML Element` within an `
 The `path` attribute is REQUIRED.
 
 ```
-EBMLFullPath             = EBMLElementOccurrence "(" EBMLReferencePath ")"
+EBMLFullPath           = EBMLElementOccurrence "(" EBMLReferencePath ")"
```

Comment:
i can't think of a better idea, unless @robUx4 wants to rename one of these components to use a shorter string length.